app/src/main/java/org/ea/sqrl/utils/EncryptionUtils.java

```diff
@@ -219,7 +219,23 @@ public static String decodeUrlSafeString(String data) throws Exception {
         return new String(decodeUrlSafe(data), "UTF-8");
     }
 
+    /***
+     * Performs a byte-by-byte XOR operation on the provided byte arrays.
+     * Since this is being used for XOR'ing cryptographic keys, in order to avoid information leakage
+     * caused by (partial) reuse of key material, this function enforces both byte arrays to have the
+     * same length (nr of bytes).
```

Comment:
Don't think we need to abbreviate (number of bytes)

Reply:
Fixed. Should be ready to merge now.

Reply:
Fixed. Should be ready to merge now!